doc/api/permissions.md

```diff
@@ -150,6 +152,12 @@ There are constraints you need to know before using this system:
 
 #### Limitations and Known Issues
 
+* The permission model implements a "seat belt" approach, which prevents trusted
```

Comment:
Shouldn't it be placed on top? Right after:

> This feature does not protect against malicious code. According to the Node.js
  [Security Policy][], Node.js trusts any code it is asked to run.

Reply:
I don't have a strong opinion. I assumed this was a temporary weakness until someone figures out a solution, but if you are saying that this is not a limitation/known issue because it is by design, then I am happy to move it up :)

Reply:
Thanks